projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.component.html

```diff
@@ -199,8 +199,8 @@
     <div class="igx-banner__message">
         <span class="igx-banner__text">
             <ng-container
-                *ngTemplateOutlet="this.crudService.row?.isAddRow ? rowAddText : resolveRowEditText || defaultRowEditText;
-                context: { $implicit: !this.crudService.row?.isAddRow ? rowChangesCount : null }">
+                *ngTemplateOutlet="this.crudService.row?.getClassName() === 'IgxAddRow' ? rowAddText : resolveRowEditText || defaultRowEditText;
```

Comment:
Why are you changing that? @skrustev preserved the isAddRow property and it should work like before.

Reply:
It's not me changing that. This approach was undertaken in https://github.com/IgniteUI/igniteui-angular/pull/9877/files#diff-9511c230760b50c458554bf036523d4de868b74f0f9fbfcb80d5428068b938caR250. I simply resolved the merge conflict which btw didn't bother that particular check, it's just that it's in the same row. For me, both of them work.